src/EnergyPlus/Boilers.cc

```diff
@@ -219,7 +219,7 @@ void GetBoilerInput(EnergyPlusData &state)
             ShowContinueError(state, format("Invalid {}={}", state.dataIPShortCut->cAlphaFieldNames(2), state.dataIPShortCut->cAlphaArgs(2)));
             // Set to Electric to avoid errors when setting up output variables
             thisBoiler.BoilerFuelTypeForOutputVariable = "Electricity";
-            thisBoiler.FuelType = Constant::AssignResourceTypeNum("ELECTRICITY");
+            thisBoiler.FuelType = Constant::ResourceType::Electricity;
```

Comment:
I saw examples of both of these in the code; I assume the latter is preferable?

Reply:
That's fine for now. @dareumnam will is working on related improvements in #9771